### PR TITLE
fix: add aria label to notification header for a11y

### PR DESCRIPTION
**Pull request by stefanoScalzo**
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/notification/notification-header/notification-header.component.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <ng-content select="[fd-notification-title]"></ng-content>
 <button
     fd-button
+    aria-label="close-notification"
```
Comment (salarenko) [inline libs/core/src/lib/notification/notification-header/notification-header.component.html:5]: Should we use kebab-case here? This is normal text value that should be read by device (at least that's how I understand it).

Comment (JKMarkowski) [reply libs/core/src/lib/notification/notification-header/notification-header.component.html:5]: Shouldn't we have a way to add it in different language?

Comment (stefanoScalzo) [reply libs/core/src/lib/notification/notification-header/notification-header.component.html:5]: fixed, the user will now decide like they do in shellbar :)

Comment (JKMarkowski) [reply libs/core/src/lib/notification/notification-header/notification-header.component.html]: Thanks Stefano, we can merge it :D

**Pull request merged**